tools/nl/embeddings/README.md

```diff
@@ -42,6 +42,8 @@ Then, you can run:
 
 This script, upon completion, should update the `palm_alternatives.csv` under the `csv` folder. Note, the API could be throttled so there are several timeouts in the scripts. That, along with the fact that the API response can be on O(seconds), this script can take a while to complete. For ~1500 SVs, this may take 1-2 hours.
 
+Note: the file `sheets/dc_nl_svs_curated.csv` should reflect the updated/new format the file downloaded from Google Sheets to manage the list of StatVars and human-curated descriptions (alternatives). However, for now it can be ignored because this migration is a work-in-progress.
```

Comment:
nit: link to the doc maybe: https://docs.google.com/spreadsheets/d/1-QPDWqD131LcDTZ4y_nnqllh66W010HDdows1phyneU?

Reply:
Not relevant anymore